user/overview.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Shouldn't this file be `README.md` in this folder? This way you don't need an additional sub-entry in the "Your Users" section. People will see this page when they click the "Your users" section header instead (currently they see an empty page).

Reply:
As discussed its set up like:
- in the README.md the user can browse available plages
- the overview is a walk through page, TO DO for the buyer and supplier processes